release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -584,6 +584,12 @@ For example, the plugin can highlight unexpected differences, such as mismatched
 You can use the `cluster-compare` plugin in development, production, and support scenarios.
 
 For more information about the `cluster-compare` plugin, see xref:../scalability_and_performance/cluster-compare/understanding-the-cluster-compare-plugin.adoc#cluster-compare-overview_understanding-cluster-compare[Overview of the cluster-compare plugin].
+[id="ocp-release-notes-node-tuning-operator-deferred-updates_{context}"]
+==== Node Tuning Operator: Deferred Tuning Updates
+
+In this release, the Node Tuning Operator introduces support for deferring tuning updates. This feature allows administrators to schedule updates to be applied during a maintenance window.
```

Comment:
```suggestion
In this release, the Node Tuning Operator introduces support for deferring tuning updates. Administrators can schedule updates to be applied during a maintenance window with is feature.
```
Per IMB style guidelines, don't use "allow" in this way. https://www.ibm.com/docs/en/ibm-style?topic=word-usage